data/observations/place_summary/construct.ru

```diff
@@ -1,4 +1,4 @@
-BASE <http://edu.ontotext.com/resource/>
+BASE <https://schools.ontotext.com/data/resource/>
```

Comment:
This change become part of this PR accidentally, it slipped in somehow ... I prefer to keep it this way, because there's a bunch of depending PRs.